.prettierignore

```diff
@@ -0,0 +1,12 @@
+# All examples, codelabs, etc.
+examples/*
```

Comment:
Do we want to format examples? They're code too.

Reply:
Yes, but in the status quo they aren't being linted. I wanted to do that in a follow up PR. I'll add it to the future work section and file issues for those.